client/src/test/java/jp/co/soramitsu/iroha/java/Example3.java

```diff
@@ -0,0 +1,154 @@
+package jp.co.soramitsu.iroha.java;
+
+import iroha.protocol.BlockOuterClass;
+import iroha.protocol.Primitive.RolePermission;
+import java.math.BigDecimal;
+import java.security.KeyPair;
+import java.util.Arrays;
+import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3;
+import jp.co.soramitsu.iroha.testcontainers.IrohaContainer;
+import jp.co.soramitsu.iroha.testcontainers.PeerConfig;
+import jp.co.soramitsu.iroha.testcontainers.detail.GenesisBlockBuilder;
+import lombok.val;
+
+public class Example3 {
+
+    private static final String bankDomain = "bank";
+    private static final String userRole = "user";
+    private static final String usdName = "usd";
+
+    private static final Ed25519Sha3 crypto = new Ed25519Sha3();
+
+    private static final KeyPair peerKeypair = crypto.generateKeypair();
+
+    private static final KeyPair useraKeypair = crypto.generateKeypair();
+    private static final KeyPair userbKeypair = crypto.generateKeypair();
+
+    private static String user(String name) {
+        return String.format("%s@%s", name, bankDomain);
+    }
+
+    private static final String usd = String.format("%s#%s", usdName, bankDomain);
+
+    /**
+     * <pre>
+     * Our initial state cosists of:
+     * - domain "bank", with default role "user" - can transfer assets and can query their amount
+     * - asset usd#bank with precision 2
+     * - user_a@bank, which has 100 usd
+     * - user_b@bank, which has 0 usd
+     * </pre>
+     */
+    private static BlockOuterClass.Block getGenesisBlock() {
+        return new GenesisBlockBuilder()
+                // first transaction
+                .addTransaction(
+                        // transactions in genesis block can have no creator
```

Comment:
I think it would be better to have code instead of comments, so maybe:
```
... creatorOfGenesisBlock = null;
```

Reply:
But I'm not Java expert, so the issue is up to You.

Similar situations are below:
```
// by default peer is listening on port 10001
```
can be:
Integer defaultPeerListeningPort = 10001

Reply:
This code is copied from Example1